test_scripts/Polices/Policies_Security/001_ATF_P_PolicyTable_GeneralSDL_behavior.lua

```diff
@@ -0,0 +1,82 @@
+----  Name of requirement that is covered.
+----- [General] Policies enough memory for PTS
+----  Policies Manager must have enough memory to hold a Policy Table Snapshot in memory.
+
+--------------------------------------------------------------------------------------------
+-----------------------------Required Shared Libraries---------------------------------------
+---------------------------------------------------------------------------------------------
```

Comment:
@Aleksandriys update script with approved header and block names from our template